tinman/txgen.py

```diff
@@ -307,15 +322,35 @@ def port_snapshot(conf, keydb, silent=True):
 
 def build_actions(conf, silent=True):
     keydb = prockey.ProceduralKeyDatabase()
-
-    start_time = datetime.datetime.strptime(conf["start_time"], "%Y-%m-%dT%H:%M:%S")
+    stats_start = datetime.datetime.utcnow()
+    stats = account_stats(conf, silent)
+    stats_elapsed = datetime.datetime.utcnow() - stats_start
+    num_accounts = stats["num_accounts"]
+    transactions_per_block = conf["transactions_per_block"]
+    
     genesis_time = datetime.datetime.utcfromtimestamp(STEEM_GENESIS_TIMESTAMP)
+    
+    # Two transactions per account (create and update).
+    predicted_transaction_count = num_accounts * 2
+    
+    # The predicted number of blocks for accounts.
+    predicted_block_count = predicted_transaction_count // transactions_per_block
+    
+    # The number of seconds required to setup transactions is a multiple of
+    # the initial time it takes to do the account_stats() call.
+    predicted_transaction_setup_seconds = (stats_elapsed.seconds * 2)
+    
+    # Pad for update witnesses, vote witnesses, clear rounds, and transaction
+    # setup processing time
+    predicted_block_count += 100 + (predicted_transaction_setup_seconds // 3)
```

Comment:
This magic 100 should be a const. Docs should be on it.

Reply:
√